main.tex

```diff
@@ -268,20 +268,22 @@ \section*{Abstract} %(301 out of 300 words)
 %3) What is the methodology being used to answer the stated research question? (81 words)
 Therefore, we exploit the provenance data model, PROV-DM, of the World Wide Web Consortium and, building on previous work, continue developing a PROV ontology for simulation studies.
 Based on a case study of 19 \wnt{}/\bcat{} signaling models, we identify crucial entities and activities as well as useful metadata to both capture the provenance information of individual simulation studies and relate these forming a family of models.
-The approach is implemented in \href{https://github.com/SFB-ELAINE/WebProv}{\webprov{}}, which allows one to insert and query provenance information.
+The approach is implemented in \href{https://github.com/SFB-ELAINE/WebProv}{\webprov{}}, a web application that allows one to insert and query provenance information.
```

Comment:
I thought giving a bit of an explanation of WebProv in the abstract would be useful

Reply:
Thanks! We need to remove 4 words. (We are at a word count of 304/300.) Do you have a suggestion?

Reply:
`allows one to` can be changed to `can`. Also above in section 2 of the abstract, `we need to identify and look` can probably just be changed to `we need to identify`

Reply:
I've changed the first one to "a web application for inserting and querying provenance information". OK? I've removed "and look at".

Reply:
Yeah sounds great!